### PR TITLE
Trivial: Remove obsolete comments from doc/CHANGELOG.

### PR DESCRIPTION
**Pull request by rec**
@vinniefalco @ximinez

### CODE REVIEW

File: doc/CHANGELOG [hunk 1]
```diff
@@ -1,20 +1,3 @@
-VFALCO NOTE - This file appears to be unmaintained.
+The file lists rippled version numbers and the pull requests they contain.
```
Comment (ximinez) [inline doc/CHANGELOG]: "...and the github.com pull requests..."

**Comment by ximinez** [comment]
Obviously correct, and doesn't change functionality anyway. The comment above is just a suggestion so that those unfamiliar with our process will know where to start. But then where else would the PRs be from?

:+1:

**Comment by rec** [comment]
I made a tiny change.

Wondering if a timestamp might be useful in future...?

**Comment by HowardHinnant** [comment]
Only timestamps in the past are useful. :-)

**Comment by rec** [comment]
Updating this, just to see what Github's API tells me about this one.  :-D

**Comment by nbougalis** [comment]
Merged as 44c5e337ab28c57f5ae6b23184c243df7a431fa8.

**Pull request closed without merging**